src/bin/cargo/commands/owner.rs

```diff
@@ -6,46 +6,137 @@ use cargo_credential::Secret;
 pub fn cli() -> Command {
     subcommand("owner")
         .about("Manage the owners of a crate on the registry")
-        .arg(Arg::new("crate").value_name("CRATE").action(ArgAction::Set))
+        .arg_quiet()
+        .arg(Arg::new("crate").hide(true))
```

Comment:
nit: move this down with the other hidden flags

Reply:
Oops, duplicated this, see https://github.com/rust-lang/cargo/pull/11879#discussion_r1158546984